delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/IndexingData.scala

```diff
@@ -1,21 +1,46 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model
 
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
 import ch.epfl.bluebrain.nexus.delta.rdf.Triple.predicate
 import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary
 import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.{rdfs, skos}
 import ch.epfl.bluebrain.nexus.delta.rdf.graph.Graph
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{ResourceF, ResourceRef}
 import io.circe.Json
 import org.apache.jena.graph.Node
 
 /**
   * ElasticSearch indexing data
   *
+  * @param id                    the resource id
+  * @param deprecated            whether the resource is deprecated
+  * @param schema                the resource schema
+  * @param types                 the resource types
   * @param selectPredicatesGraph the graph with the predicates in ''graphPredicates''
   * @param metadataGraph         the graph with the metadata value triples
   * @param source                the original payload of the resource posted by the caller
   */
-final case class IndexingData(selectPredicatesGraph: Graph, metadataGraph: Graph, source: Json)
+final case class IndexingData(
```

Comment:
This `IndexingData` seems to be the same in blazegraph and elasticsearch. Why don't we move it to `sdk-views` module? The only different I can spot is the `Json`, which can be ignored by using the correct apply method